src/frontend/context/SimuleringContext.tsx

```diff
@@ -24,6 +24,7 @@ const [SimuleringProvider, useSimulering] = constate(({ åpenBehandling }: IProp
         status: RessursStatus.HENTER,
     });
     const { toggles } = useApp();
+    const maksLengdeFritekst = 1500;
```

Comment:
Skulle det ikke være lov å sende inn fritekst lengre enn 1500? Er det noe oppgave på det? 
For validering virker 1500 tegn veldig mye 😆